packages/sakuli-legacy/src/context/sahi/accessor/accessor-util.class.ts

```diff
@@ -303,7 +326,7 @@ export class AccessorUtil {
         : "^\\s*" + str.replace(/[.*+?^${}()|[\]\\]/g, "\\$&") + "\\s*$"
     );
     this.testExecutionContext.logger.debug(
-      `Converted string "${str}" to regex ${regex}`
+      `Converted identifier string "${str}" to regex ${regex}`
     );
```

Comment:
IMHO, the whole message does not add any value, even in debugging. You just get the info that a string was wrapped into `/^\s* \s*$/` programmatically. You are not able to use this regex in a simple way to receive the same results as sakuli would receive. Maybe this is a candidate to be moved to trace level.